File: app/src/main/java/io/github/anvell/stackoverview/model/QuestionsResponse.java
package io.github.anvell.stackoverview.model;

import com.google.gson.annotations.SerializedName;

public class QuestionsResponse extends ResponseBase<Question> {

    @SerializedName("has_more")
    public Boolean hasMore;

    @SerializedName("quota_max")
    public int quotaMax;

    @SerializedName("quota_remaining")
    public int quotaRemaining;
}

File: app/src/main/java/io/github/anvell/stackoverview/model/QuestionsResponse.kt
package io.github.anvell.stackoverview.model

import com.google.gson.annotations.SerializedName

class QuestionsResponse(
    items: List<Question> = listOf(),
    @SerializedName("has_more") val hasMore: Boolean = false,
    @SerializedName("quota_max") val quotaMax: Int = 0,
    @SerializedName("quota_remaining") val quotaRemaining: Int = 0
) : ResponseBase<Question>(items)